src/App.js

```diff
@@ -17,6 +17,20 @@ import { computeBadges } from './badgeLogic';
 import { timestampsOk } from './constants';
 import { Row } from './row';
 import BootstrapTable from 'react-bootstrap-table-next';
+import alliance from './icons/alliance.png';
```

Comment:
It looks like it's a lot imports, we should be able to import all icons (all exported within another file). What do you think?